client/client.go

```diff
@@ -145,6 +158,37 @@ func New(ctx context.Context, address string, opts ...ClientOpt) (*Client, error
 		gopts = append(gopts, grpc.WithStreamInterceptor(grpc_middleware.ChainStreamClient(stream...)))
 	}
 
+	kaTimeDuration := defaultClientKeepaliveTime
+	if timeEnv := os.Getenv("OKTETO_KEEPALIVE_CLIENT_TIME_MS"); timeEnv != "" {
+		timeDuration, err := time.ParseDuration(fmt.Sprintf("%sms", timeEnv))
+		if err != nil {
+			kaTimeDuration = defaultClientKeepaliveTime
+		}
+
+		kaTimeDuration = timeDuration
+	}
+
+	kaTimeoutDuration := defaultClientKeepaliveTimeout
+	if timeoutEnv := os.Getenv("OKTETO_KEEPALIVE_CLIENT_TIMEOUT_MS"); timeoutEnv != "" {
+		timeoutDuration, err := time.ParseDuration(fmt.Sprintf("%sms", timeoutEnv))
+		if err != nil {
+			kaTimeoutDuration = defaultClientKeepaliveTimeout
+		}
+
+		kaTimeoutDuration = timeoutDuration
```

Comment:
Same as above, we are always overriding the value

Reply:
yes, thanks @AgustinRamiroDiaz